### ISSUE TITLE
how do you turn on random generator if it is turned off for performance.

### ISSUE THREAD

**Issue by WarrenLE** [wonderunit/storyboarder#743]

**Comment by CuberToy** [comment]
It's not "turn off for performance" per say. In fact, the generator use WebGL tech. If it's turned off, then your GPU is probably not WebGL compatible. There's no way to turn it on if that's the case.

You should post your GPU model though, just to be sure.

**Comment by WarrenLE** [comment]
It in a MacBook Pro.  Radeon HD 6770M

**Comment by CuberToy** [comment]
Ok so it's weird cause I see it as WebGL compatible. Plus, the browser pretty much do all the job for all I know, and Storyboarder is built on Chromium/Electron.

Does this work when you open it in your browser ? https://get.webgl.org/

If yes, then I would guess (but I'm no dev) that chromium as flag off webgl for some reason (I've seen some people having the issue on chrome using the same setup)

**Comment by WarrenLE** [comment]
It works with that link.  I see the spinning cube.

**Comment by CuberToy** [comment]
Ok. So I hope the devs will answer (I kinda try, but sometimes I can't help, really sorry). It looks like a legitimate issue. Don't know... #692 might give some clues.

Though I didn't find it on any Chromium blacklisting and everything tells me it's WebGL compatible, the card itself is +5yrs old so... yeah, might be an issue with the most recent tech, particulary the shading.

Did you open the link on Chrome? Just curious here.

**Comment by WarrenLE** [comment]
I clicked this link * https://get.webgl.org/) and it launched Safari and played the spinning cube saying WebGL enabled.  I don't use Chromium web browser if that it what you are asking.

**Comment by CuberToy** [comment]
Yeah I kinda guessed. What I meant was that the browser Chrome, is based on Chromium... so is Storyboarder. So maybe Chromium blacklisted your GPU or something similar. Safari and others don't blacklist, if I remember correctly, so it may work on Safari (as your card is definitely WebGL compatible) but not on Chromium based software (browser, or others like Storyboarder). Probably because although the card is webGL compatible, it's not with the latest version of WebGL (which Storyboarder use, for more complex shading for exemple). Not much you can do, heck even not much the devs can do. But there's hope, as some workaround are available. That's why I'm really hoping they can respond to this.
